extra-creativity/darktable/autobuild/defines

```diff
@@ -15,6 +15,7 @@ CMAKE_AFTER="-DBINARY_PACKAGE_BUILD=1 \
              -DUSE_LUA=ON \
              -DUSE_COLORD=ON \
              -DRAWSPEED_ENABLE_LTO=ON \
+             -DCMAKE_SKIP_RPATH=OFF \
```

Comment:
And why is this?

Reply:
So that we no longer need to ship global `/etc/ld.so.conf.d/darktable.conf`

Reply:
Got it.